packages/website/src/showcases/showcases.json

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
Since we're now completely generating this file, it should probably be added to the `.gitignore` file, like the kit pages are.